.changes/unreleased/Under the Hood-20221004-105758.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Under the Hood
```

Comment:
Format the commit comment to be less than `80` chars for readability.

Reply:
The commit comment just contains some (optional) additional details. I suppose it is good as is?